src/EventGrid/EventGrid/ChangeLog.md

```diff
@@ -27,6 +27,21 @@
   * IsNullOrUndefined
   * IsNotNull
 
+## Version 1.50
```

Comment:
1. No need to update the version manually. Our tools will do it when build the release.
1. Just add changelog to section `Upcoming Release`

Reply:
Removed manual version increment and added changelog to the `Upcoming Release` section